### DISCUSSION TITLE
Daily Challenge might be improved

### DISCUSSION THREAD

**Discussion by Myazusa** [ppy/osu#29077] [Ideas]
"**Daily Challenge**" is an innovative attempt by osu!lazer. However, I believe that the gameplay has great potential and could perhaps be the beginning of a PVE mode. Currently, players can gain rankings and records through "**Daily Challenges**", which doesn't seem very meaningful.

As a derivative gameplay of "**Daily Challenge**", it could perhaps be designed like this:

1. First, segment players by score brackets. Players in each bracket can earn a different number of items , it's similar to "**Boss Challenge Attempts**" from the "**Daily Challenge**" each day.

![1](https://github.com/user-attachments/assets/ee9a7e63-0522-4417-b16c-1ec8a433ec72)

2. On weekends, open a song challenge similar to "**World Boss**". This mode requires "**Boss Challenge Attempts**" to participate. The "**World Boss Song**" should be assigned a reasonable "**Health**", which is shared by all players. When each player challenges this song and scores, a certain amount of "**Health**" is deducted upon settlement.

   In the image, green represents the "**World Boss Song**" and red represents the "**Health**".

![2](https://github.com/user-attachments/assets/93f7a68e-d556-411d-a7c1-5eb26c30da3d)

3. During the "**World Boss Song**" challenge, there's also a low probability of dropping rewards such as achievements, avatar frames and background frames. On the next Monday, players will receive "**Contribution Points**" based on the number of times they participated and the amount of "**Health**" they reduced, according to a certain algorithm. If the "**World Boss Song**"'s "**Health**" drops to 0, all players receive additional "**Contribution Points**".

4. "**Contribution Points**" can be exchanged for some interesting things such as avatar frames, background frames and titles on the osu! official website's "**Contribution Shop**" page.

![3](https://github.com/user-attachments/assets/269aac9c-413e-4837-b3d3-cc4658cbb04e)

5. A new world rankings can be added for this gameplay. Players can team up to participate in the "**World Boss Song**", and rewards will be increased when teaming up (a gameplay similar to multiplayer).

These are some of my personal suggestions. I hope the developers can add more innovative gameplay features to make the game more interesting. Thank you!

**Comment by marioman63** [comment]
I would be very concerned about giving this mode any sort of prize like that. would feel very very dirty for this game. I come to osu! specifically because it doesn't have mobile game mechanics in it. Why do we need prizes for "winning" the challenge anyways?

**Reply by Myazusa** [reply]
You are right. To keep the game clean, this reward should be similar to the "**osu! achievement**" system. It's merely a motivation for players to play, not to differentiate players or for any other purpose.
